spec/smart_app_launch/client_suite/access_alcs_interaction_test_spec.rb

```diff
@@ -0,0 +1,161 @@
+RSpec.describe SMARTAppLaunch::SMARTClientAccessAppLaunchConfidentialSymmetricInteraction, :request do # rubocop:disable RSpec/SpecFilePathFormat
+  let(:suite_id) { 'smart_client_stu2_2' }
+
+  describe 'during the acess wait test' do
+    let(:static_uuid) { 'f015a331-3a86-4566-b72f-b5b85902cdca' }
+    let(:test) { described_class }
+    let(:alcs_reg_test) { suite.children[0].children[1] } # app launch confidential symmetric reg test
+    let(:test_session) do # overriden to add suite options
+      repo_create(
+        :test_session,
+        suite: suite_id,
+        suite_options: [Inferno::DSL::SuiteOption.new(
+          id: :client_type,
+          value: SMARTAppLaunch::SMARTClientOptions::SMART_APP_LAUNCH_CONFIDENTIAL_SYMMETRIC
+        )]
+      )
+    end
+    let(:results_repo) { Inferno::Repositories::Results.new }
+    let(:requests_repo) { Inferno::Repositories::Requests.new }
+    let(:patient_id) { '999' }
+    let(:patient_read_url) { "/custom/#{suite_id}/fhir/Patient/#{patient_id}" }
+    let(:patient_read_response) { %({ "resourceType": "Patient", "id": "#{patient_id}" }) }
+    let(:fhir_read_resources_bundle) do
+      %({ "resourceType": "Bundle", "entry": [ { "resource": #{patient_read_response} } ] })
+    end
+    let(:token_url) { "/custom/#{suite_id}#{SMARTAppLaunch::TOKEN_PATH}" }
+    let(:token_endpoint) { 'https://inferno.healthit.gov/suites/custom/smart_client_stu2_2/auth/token' }
+    let(:authorization_url) { "/custom/#{suite_id}#{SMARTAppLaunch::AUTHORIZATION_PATH}" }
+    let(:authorization_code_token) { described_class.client_id_to_token(client_id, 10) }
+    let(:client_id) { 'test_client' }
+    let(:smart_client_secret) { 'secret' }
+    let(:redirect_uri) { "https://inferno.healthit.gov/redirect"}
+    let(:token_request_body_valid) do
+      { grant_type: 'authorization_code',
+        scope: 'system/*.rs'
+      }
+    end
+
+    describe 'it responds to token requests' do
+
+      it 'it succeeds when providing the client secret' do
+        
+        # run reg test to save smaart_jwk_set input
```

Comment:
```suggestion
        # run reg test to save smart_jwk_set input
```
This typo appears a bunch of times.

Reply:
fixed all in [742af51](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/96/commits/742af51aa15e06b30cc094a010658d3b287e036e)